### PR TITLE
Added a note on how to contribute in the README

### PR DESCRIPTION
**Pull request by phillip-kruger**
Signed-off-by:Phillip Kruger <phillip.kruger@gmail.com>

As discussed in the community meeting, making "How to contribute" more visivle

### CODE REVIEW

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

File: README.adoc [hunk 1]
```diff
@@ -20,6 +20,22 @@ image:https://badges.gitter.im/eclipse/microprofile-reactive.svg[link="https://g
...
 
+'''
+== How to contribute
+
+Do you want to contribute to this project? Here is what you can do:
+
+* Fork the repository, make changes, then do a pull request.
+* https://github.com/eclipse/microprofile-reactive-streams-operators/issues[Create or fix an issues].
+* https://gitter.im/eclipse/microprofile-reactive[Join us on Gitter to discuss this project].
+* Join our https://calendar.google.com/calendar/embed?src=gbnbc373ga40n0tvbl88nkc3r4%40group.calendar.google.com[bi-weekly meeting] every second Tuesday at https://www.timeanddate.com/time/map/[10h00 GMT].
```
Comment (cescoffier) [inline README.adoc]: The reactive hangouts are every week. So probably the wrong link.

File: README.adoc [hunk 2]
```diff
@@ -20,6 +20,22 @@ image:https://badges.gitter.im/eclipse/microprofile-reactive.svg[link="https://g
 
 = MicroProfile Reactive Streams Operators
 
+'''
+== How to contribute
+
+Do you want to contribute to this project? Here is what you can do:
+
+* Fork the repository, make changes, then do a pull request.
```
Comment (cescoffier) [inline README.adoc]: Should it mention:

* Sign the Eclipse CLA
* Sign-off all your commit

Comment (phillip-kruger) [reply README.adoc]: Good point, let me add that quickly

File: README.adoc (hunk 1 above)
Comment (phillip-kruger) [reply README.adoc]: Fixed. Will commit soon.

Comment (phillip-kruger) [reply README.adoc]: Done

File: README.adoc (hunk 2 above)
Comment (phillip-kruger) [reply README.adoc]: Done

**Comment by cescoffier** [comment]
Thanks!

**Pull request merged**